src/test/kotlin/org/openrewrite/java/testing/junit5/TemporaryFolderToTempDirTest.kt

```diff
@@ -235,10 +262,10 @@ class TemporaryFolderToTempDirTest : JavaRecipeTest {
                 File tempDir1;
             
                 File subDir = newFolder(tempDir1, "sub");
-                File subDirs = newFolder(tempDir1, "foo", "bar", "baz");
+                File subDirs = newFolder(tempDir1, "foo/bar/baz");
```

Comment:
What if the arguments aren't String literals but identifiers or method invocations?

Reply:
nice catch @sambsnyd.  The subdirectory would reflect the identifier names.